README.md

```diff
@@ -10,6 +10,73 @@ Flatland Benchmarks (FAB) is an open-source web-based platform for running Bench
+| Digital Environment  | E.g. Flatland, Grid2Ops.                                                                                                                  |
+| AI Agent             | E.g. a Flatland 3 competition solution taking actions on the Flatland environment based on observations and rewards from the environment. |
+| Evaluation Submodule | E.g. FAB Flatland 3 evaluator.                                                                                                            |
+| UI                   | E.g. Interactive AI Frontend                                                                                                              |
+
+## Data Model
+
+Inspired by [LIPS](https://github.com/IRT-SystemX/LIPS), high-level data model is as follows:
+
+![DataModel.drawio.png](docs/img/readme/DataModel.drawio.png)
+
+In words:
+
+* The simulation engine produces run information/measurements, referring to a scenario. The exact contents are domain-specific. In the railway domain, these are the actions taken by the action and the rewards given by the environment. In the power grid case, these are observations and predictions.
+* The evaluation submodule has an evaluation. It may also have scenario-specific metadata (e.g. ground truth for prediction into the future problems). In the railway domain, the evaluation module aggregates the rewards along different simulation steps and episodes.
+* The output of the evaluation submodule is in a generic form referring to a scenario and the simulation engine run: it contains metrics values grouped by KPI.
+* Validation campaigns can be defined in the FAB hub: weights and thresholds are defined per objective, KPI and/or for individual metrics and per KPI. An overall threshold for the success of the validation campaign is defined.
+
+| Level               | Validation                                                                                                                           |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| Validation Campaign | <ul><li>succeeded yes or no (iff all domain validation campaigns succeed)</li></ul>                                                  |
+| Objective           | <ul><li>succeeded yes or no (iff threshold met)</li><li>diff to threshold</li><li>weighted value</li><li>...</li></ul>               |
+| KPI                 | <ul><li>succeeded yes or no (iff threshold met)</li><li>diff to threshold</li><li>weighted value</li><li>...</li></ul>               |
+| Metrics             | <ul><li>succeeded yes or no (iff threshold met)</li><li>diff to threshold</li><li>weighted value</li><li>value</li><li>...</li></ul> |
+
 📦 TL; DR;
 ----------
 To participate, or even organize your own benchmarks or competitions, you don't need to install anything, you just need to sign in an instance of the platform (
```

Comment:
line break before the opening parenthesis or after the closing parenthesis ?
(see the blank in the rendered markdown)